demonstrations/tutorial_tn_basics.py

```diff
@@ -0,0 +1,578 @@
+r"""Tutorial: ABC of tensor networks for quantum circuits
+=============================================================
+
+If you are well-versed in the topics of quantum computing or quantum information, chances are you have heard (maybe a lot) about tensor networks. In fact, tensor networks are a rather widespread "tool" with applications ranging across physics, math, and computer science.
+
+Part of the excitement surrounding tensor networks is due to their ability to represent complex data efficiently, which allows for—among other things—fast classical simulations. In addition, the diagrammatic language accompanying tensor networks makes working with them intuitive and suitable for describing a vast range of mathematical concepts, including quantum circuits.
+
+The :ref:`first part of this tutorial <part_one>` introduces the basic tensor network notions and definitions to quantum practitioners who are familiar with quantum computing but new to the tensor networks world.
+
+Then, building on this introduction, in the :ref:`second part <part_two>` we explore topics aimed at more seasoned readers. Check this section out if you want to understand how tensor networks and quantum circuits are related!
+
+Without further ado, let’s dive right in! 🤓📚
+
+.. _part_one:
+
+Part 1: A first glimpse into the tensor networks world
```

Comment:
I don't think separating into two parts is necessary (in the section title). It does not follow consistent format with other demos on the PennyLane site.

```suggestion
A first glimpse into the tensor networks world
```